src/containers/Signin/SigninContainer.js

```diff
@@ -0,0 +1,14 @@
+import {connect} from 'react-redux';
+import Signin from '../../components/Signin';
+import {fetchSigninToken} from './signinActions'
+
+const mapStateToProps = (state) => {
+    return state;
```

Comment:
You don't have to pass entire state here. As sing in /sign up doesn't require remainders and all.